Add metric for learned ports

grafnu opened the pull request.

anarkiwi commented:
Please add a test.

anarkiwi commented:
It also looks like associations will not age out (e.g if MAC moves ports or IP changes the old association will still be present).

grafnu commented:
Updated, PTAL. Test added, expiry added, and removed the l3 association since it's fundamentally l2 learning. Handles the moved case since there's only one port per dp/mac/vlan, so a move overwrites the previous value.
